vue.eik.js

```diff
@@ -1,10 +1,6 @@
-import * as eik from '@eik/esbuild-plugin';
 import esbuild from 'esbuild';
 
-await eik.load();
-
 await esbuild.build({
-  plugins: [eik.plugin()],
```

Comment:
Why was this removed? Now in the output icons.js we have `import sth from 'react'` instead of an eik url 🤔 Was that intended?

Reply:
Since this is an npm package being used in project that are either `react` or `vue`, they are already importing react or vue, and probably bundling their code. Eik in that project would take care of that. Doing this here was breaking icons in multiple apps because esbuild doesn't like packages importing from `http:` :)